controllers/gameReviewsController.ts

```diff
@@ -0,0 +1,32 @@
+import { RequestHandler } from "express";
+
+import { PrismaClient } from "@prisma/client";
+import { create } from "domain";
+const prisma = new PrismaClient();
+
+export const addReview: RequestHandler = async (req, res) => {
```

Comment:
add req and res types

Reply:
Types added. Will not miss types for req res again.